api/src/auth.py

```diff
@@ -63,6 +63,9 @@ def login():
 
     if not bcrypt.checkpw(password.encode('utf-8'), user['hashed']):
         return jsonify({'msg': 'Bad username or password'}), 401
+    #if (user["email_confirmed"] == False):
+        #print("not allowed")
+        #return jsonify({'msg': 'Email not confirmed'}), 403
```

Comment:
We should probably allow them to login even they didn't confirm their email, otherwise they would be able to resend confirmation email

Reply:
It does allow user to log in already, I dont think i changed this